### PR TITLE
Delete user profile 2

### PR DESCRIPTION
**Pull request by aeydie**
I created a new branch off of main to hope for less merge requests.  Button will pop up once a user profile has been created as an option on the user profile to delete user.  If button is clicked, user profile is deleted and the user is directed back to the events page.

### CODE REVIEW

File: src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java [hunk 1]
```diff
@@ -9,4 +9,5 @@
 @Repository
 public interface UserRepository extends CrudRepository<User, Integer> {
     Optional<User> findByUsername(String username);
+    Optional<User> deleteByUsername(String username);
```
Comment (lfallert) [inline src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java]: Do we need this method?

**Review by lfallert — APPROVED** [review]
works as intended

**Review by CNichole726 — APPROVED** [review]
works!

File: src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java (hunk 1 above)
Comment (aeydie) [reply src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java]: I don't think so and I think you already removed it - I'll make sure :)

**Pull request merged**